demo/app.component.ts

```diff
@@ -61,6 +61,32 @@ export class AppComponent implements OnInit {
     }
   ];
 
+  layout: string = 'dagre';
+  layouts: any[] = [
+    {
+      label: 'Dagre',
+      value: 'dagre',
+    },
+    {
+      label: 'Dagre Nodes Only',
+      value: 'dagreNodesOnly',
+    },
+    {
+      label: 'Dagre Cluster',
+      value: 'dagreCluster',
+      isClustered: true,
+    },
+    {
+      label: 'Cola Force Directed',
+      value: 'colaForceDirected',
```

Comment:
Selecting these raises an error in the demo page, because we don't import them in the LayoutService any more. They would need to be imported and instantiated in the demo app, which would also be a good way to demo using custom layouts.